app/views/shared/menu/_large_menu.html.haml

```diff
@@ -43,6 +43,14 @@
           = t 'powered_by'
           %a{href: '/'}
             = t 'title'
+      - if I18n.available_locales.count > 1
+        %li.language-switcher.has-dropdown.not-click
```

Comment:
I see that you copied this from the cog symbol, but I can't figure out what the `not-click` class does here. I can't find it in the documentation. This comment says it's only for internal use: https://stackoverflow.com/questions/21350494/foundation-5-top-bar-doesnt-drop-down#comment38329499_23280375